newIDE/app/src/InstancesEditor/StatusBar.js

```diff
@@ -41,22 +41,41 @@ export default class StatusBar {
   }
 
   render() {
-    const padding = 5;
-    const borderRadius = 4;
+    const textPadding = 5;
+    const statusBarPadding = 20;
+    const borderRadius = 6;
+    const textXPosition = Math.round(
```

Comment:
Doing this, you put the text at the **right** bottom corner although it was at the left bottom corner until now. Was it a requested feature?

Reply:
And one could think that it's badly positioned since it goes too much into the scene. Is it possible to shift the text when the scrollbars appear?

Reply:
Not really, with the scrollbar added, it was now in the way. 
And as we moved it, Luni & I felt it was more natural to be at the intersection of both scrollbars.
That can be changed if we feel this is not wise

Reply:
I've pushed it as far as possible. On my screen it looks fine but lmk if you think we should do something else.
I think that making it move depending on scrollbar visibility will be too confusing as it may happen very often?